src/features/crafts/craftsSlice.js

```diff
@@ -28,16 +28,51 @@ export const fetchCrafts = createAsyncThunk('crafts/fetchCrafts', async () => {
                 priceRUB
                 trader {
                   name
+                  normalizedName
                 }
               }
               buyFor {
                 source
+                vendor {
+                  name
+                  normalizedName
+                  __typename
+                  ...on TraderOffer {
+                      trader {
+                          id
+                          name
+                          normalizedName
+                      }
+                      minTraderLevel
+                      taskUnlock {
+                          id
+                          name
+                      }
+                  }
+                }
                 price
                 priceRUB
                 currency
               }
               sellFor {
                 source
+                vendor {
+                  name
+                  normalizedName
+                  __typename
```

Comment:
What is this `__typename` property? I have no seen it before and its naming structure is different from the rest

Reply:
it's part of the graphql spec for resolving unions and interfaces. Allows consumers to know which type out of all the possible options was resolved.